Snakefile

```diff
@@ -553,8 +556,22 @@ rule post_masked_consensus_and_summary_stats_to_id3c:
         except HTTPError as http_err:
             file.write(str(http_err))
 
+            slack_data = { "text":
+                f":rotating_light: @{getpass.getuser()} Assembly failed to upload to ID3C.\nMore details at `{log}`"
```

Comment:
The local user won't always match the Slack user, for what that's worth. Nice to have when it does, though.

Reply:
Could the Slack message include some bit of the HTTP error? That might help fasting troubleshooting.